### PR TITLE
License_plate_detection research

### PR DESCRIPTION
**Pull request by Nike2002**

### CODE REVIEW

**Review by ThomasintAnker — DISMISSED** [review]

**Comment by ThomasintAnker** [comment]
Ivm tijd approved, heb ook ff de merge conflict gefixt

File: Documentation/RD_SpeedCamera/content/license_plate_recognition.tex [hunk 1]
```diff
@@ -1,4 +1,239 @@
...
+        
+            The fourth step is to create a cascade classifier using the trained Adaboost classifier. 
+            The cascade classifier consists of multiple stages, where each stage contains a set of classifiers. 
+            The output of each stage is a binary decision that determines whether the object is present or not. 
+            The cascade classifier is used to reduce the number of false positives by quickly rejecting the input that is not likely to contain the object.
+
+        \end{enumerate}
+
+    \subsection{HOG + Linear SVM}
+    The Histogram of Oriented Gradients (HOG) is a feature extraction technique used for object detection in computer vision.
```
Comment (maxlintelo) [inline Documentation/RD_SpeedCamera/content/license_plate_recognition.tex:107]: voeg HOG toe aan abbreviation in introduction.tex

File: Documentation/RD_SpeedCamera/content/license_plate_recognition.tex [hunk 2]
```diff
@@ -1,4 +1,239 @@
...
+            The output of each stage is a binary decision that determines whether the object is present or not. 
+            The cascade classifier is used to reduce the number of false positives by quickly rejecting the input that is not likely to contain the object.
+
+        \end{enumerate}
+
+    \subsection{HOG + Linear SVM}
+    The Histogram of Oriented Gradients (HOG) is a feature extraction technique used for object detection in computer vision. 
+    HOG computes the distribution of edge orientations in an image and is an effective feature descriptor for object detection.
+
+    Linear Support Vector Machines (SVMs) are a type of supervised learning algorithm that can classify input data into one of two categories based on a set of features.
```
Comment (maxlintelo) [inline Documentation/RD_SpeedCamera/content/license_plate_recognition.tex:110]: abbreviation in introduction.tex

File: Documentation/RD_SpeedCamera/content/license_plate_recognition.tex [hunk 3]
```diff
@@ -1,4 +1,239 @@
...
+        
+            Once the Linear SVM is trained, the next step is to use it for object detection in test images. 
+            The HOG feature extraction process is applied to a sliding window of the test image at multiple scales and positions.
+        
+            At each window position, the HOG features are computed, and the Linear SVM is used to classify the window as either containing an object or not. 
+            If the window is classified as containing an object, it is considered a detection.
+
+            \item \textbf{Non-Maximum Suppression}
+        
+            Finally, the algorithm performs Non-Maximum Suppression (NMS) to remove duplicate detections and retain only the most confident detections.
```
Comment (maxlintelo) [inline Documentation/RD_SpeedCamera/content/license_plate_recognition.tex:145]: abbreviation in introduction.tex

File: Documentation/RD_SpeedCamera/content/license_plate_recognition.tex [hunk 4]
```diff
@@ -1,4 +1,239 @@
...
+
+            \item \textbf{Non-Maximum Suppression}
+        
+            Finally, the algorithm performs Non-Maximum Suppression (NMS) to remove duplicate detections and retain only the most confident detections. 
+            NMS involves selecting the detection with the highest confidence score and removing all other detections that overlap with it by more than a certain threshold.
+
+        \end{enumerate}
+
+    \subsection{YOLO}
+    YOLO is an acronym for 'You Only Look Once'. It is an algorithm that can detect and recognize various objects.
```
Comment (maxlintelo) [inline Documentation/RD_SpeedCamera/content/license_plate_recognition.tex:151]: abbreviation in introduction.tex

File: Documentation/RD_SpeedCamera/content/license_plate_recognition.tex [hunk 5]
```diff
@@ -1,4 +1,239 @@
...
+            Finally, the algorithm performs Non-Maximum Suppression (NMS) to remove duplicate detections and retain only the most confident detections. 
+            NMS involves selecting the detection with the highest confidence score and removing all other detections that overlap with it by more than a certain threshold.
+
+        \end{enumerate}
+
+    \subsection{YOLO}
+    YOLO is an acronym for 'You Only Look Once'. It is an algorithm that can detect and recognize various objects. 
+    YOLO performs object detection as a regression problem and provides the class probabilities of the detected images.
+
+    The YOLO algorithm employs convolutional neural networks (CNN) to detect objects in real-time.
```
Comment (maxlintelo) [inline Documentation/RD_SpeedCamera/content/license_plate_recognition.tex:154]: abbreviation in introduction.tex

File: Documentation/RD_SpeedCamera/content/license_plate_recognition.tex [hunk 6]
```diff
@@ -1,4 +1,239 @@
...
+            \item The \(bx\) and \(by\) values represent the x and y coordinates of the center of the bounding box with respect to the enveloping grid cell.
+            \item The \(bh\) and \(bw\) values represent the height and width of the bounding box with respect to the enveloping grid cell.
+            \item The \(c1\) and \(c2\) values correspond to the different classes of objects to be detected. In the example of a soccer game, c1 might represent the class "player," and c2 might represent the class "ball." YOLO can detect as many classes as required for a given use case.
+            \end{itemize}
+
+            \item \textbf{Intersection Over Unions or IOU}
+        
+            When detecting an object in an image using YOLO, multiple grid boxes can overlap with the object, even though not all of them are relevant. 
+            The IOU (Intersection over Union) metric is used to filter out irrelevant grid boxes and retain only those that are relevant.
```
Comment (maxlintelo) [inline Documentation/RD_SpeedCamera/content/license_plate_recognition.tex:190]: abbreviation in introduction.tex

File: Documentation/RD_SpeedCamera/content/license_plate_recognition.tex [hunk 7]
```diff
@@ -1,4 +1,239 @@
...
+            Setting an IOU threshold alone may not always be sufficient when detecting objects in an image using YOLO, as an object can have multiple bounding boxes with an IOU value beyond the threshold. 
+            Keeping all of these boxes can result in noise and reduce the accuracy of the detection.
+        
+            To address this issue, YOLO employs Non-Maximum Suppression (NMS), which selects only the bounding box with the highest probability score of detection for each object. 
+            This approach ensures that only the most likely bounding box is retained for each object, while discarding the others with lower probability scores.
+
+        \end{enumerate}
+
+    \subsection{SSD (Single Shot MultiBox Detector)}
+    SSD is a real-time object detection algorithm that can detect multiple objects in an image using a single pass through a deep neural network.
```
Comment (maxlintelo) [inline Documentation/RD_SpeedCamera/content/license_plate_recognition.tex:208]: abbreviation in introduction.tex

**Review by ThomasintAnker — APPROVED** [review]

**Pull request merged**